test/acceptance/tests.py

```diff
@@ -1201,6 +1263,30 @@ def test_feedback_only(self):
         self.staff_area_page.verify_learner_final_score("Final grade: 1 out of 1")
 
 
+@ddt.ddt
+class MultipleOpenAssessmentTest(OpenAssessmentTest, MultipeOpenAssessmentMixin):
+    """
+    Test the multiple peer-assessment flow.
+    """
+
+    def setUp(self):
+        super(MultipleOpenAssessmentTest, self).setUp('multiple_ora')
+        self.staff_area_page = StaffAreaPage(self.browser, self.problem_loc)
```

Comment:
@efischer19 Thanks for taking a look at this. And @andy-armstrong too :)

They are being differentiated using their vertical-index number. 

As I have mentioned the comment, each problem has a vertical-index number, so initial problem has `vert-0` class and second one has `vert-1` and so on. So when I move on to next problem, I reset the the vert-index number of the pages that test is using.